devportalservice/testdevice.go

```diff
@@ -0,0 +1,51 @@
+package devportalservice
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/bitrise-io/go-utils/pathutil"
+)
+
+// TestDevice ...
+type TestDevice struct {
+	ID     int `json:"id"`
+	UserID int `json:"user_id"`
+	// DeviceID is the Apple device UDID
+	DeviceID   string    `json:"device_identifier"`
+	Title      string    `json:"title"`
+	CreatedAt  time.Time `json:"created_at"`
+	UpdatedAt  time.Time `json:"updated_at"`
+	DeviceType string    `json:"device_type"`
+}
+
+// ParseTestDevicesFromFile ...
+func ParseTestDevicesFromFile(path string, currentTime time.Time) []TestDevice {
```

Comment:
Shouldn't we propagate errors to the caller and let them decide how to handle it?

Reply:
That is a good point. I will change that.